roles/containers/defaults/main.yml

```diff
@@ -25,8 +25,14 @@ secret_key_base: "{{ lookup('community.general.random_string', base64=True, leng
 access_token_secret: "{{ lookup('community.general.random_string', base64=True, length=64) }}"
 refresh_token_secret: "{{ lookup('community.general.random_string', base64=True, length=64) }}"
 web_admin_username: admin
-enable_alerting: "false"
 enable_api_key: "true"
 enable_charts: "true"
+enable_alerting: "false"
+smtp_server: ""
+smtp_port: ""
+smtp_user: ""
+smtp_password: ""
+alert_sender: "alerts@trento-project.io"
+alert_recipient: "admin@trento-project.io"
```

Comment:
I wouldn't put such a default. I would let it empty. I don't think nobody has `admin@trento-project.io` as their mail

Reply:
Sure we can let them empty!
I was just reusing the same default values we use in [trento web configs ](https://github.com/trento-project/web/blob/main/config/config.exs#L35):+1: